src/main/java/io/cdap/plugin/http/source/common/BaseHttpSourceConfig.java

```diff
@@ -127,6 +128,12 @@ public abstract class BaseHttpSourceConfig extends BaseHttpConfig {
   @Macro
   protected String format;
 
+  @Macro
```

Comment:
Add field for enable quote values

Reply:
Let's have a discussion on this i believe this should be a separate feature raised in a different PR, as both schema detection and quoted values can work independently.